internal/utils/io.go

```diff
@@ -1,9 +1,10 @@
-package internal
+package utils
```

Comment:
https://blog.golang.org/package-names
We should avoid packages like utils. (We have one in the project now, but we want to refactor it)

Reply:
Renamed this package.